third_party/move/move-compiler-v2/src/pipeline/ability_processor.rs

```diff
@@ -684,9 +684,9 @@ impl<'a> Transformer<'a> {
         temp: TempIndex,
     ) -> Vec<(Loc, String)> {
         if let Some(info) = self.live_var.get_info_at(code_offset).after.get(&temp) {
-            info.usages
-                .iter()
-                .map(|loc| (loc.clone(), "used here".to_owned()))
+            info.usage_locations()
+                .into_iter()
+                .map(|loc| (loc, "used here".to_owned()))
```

Comment:
1. This is only invoked only when an error needs to be reported.
2. This is unrelated to this PR, in that this PR is not changing the previous behavior.
3. This is not a trivial change, due how this information gets propagated (eventually, these strings are passed to `diag_with_primary_and_labels`, which requires labels with locations and owned strings.

Because of all these reasons, I am keeping the code here as is.

Reply:
If it's really only used in the error case, no problem.  I thought maybe it was being stored in dataflow objects.